mssqlcli/__init__.py

```diff
@@ -1 +1,638 @@
 __version__ = '0.10.0'
+
+
+import click
+import datetime as dt
+import functools
+import humanize
+import logging
+import os
+import threading
+import traceback
+
+from codecs import open
+from collections import namedtuple
+from time import time
+
+from mssqlcli.config import (
+    get_casing_file,
+    config_location,
+    ensure_dir_exists,
+    get_config)
+
+from mssqlcli import mssqlutils
+from mssqlcli.completion_refresher import CompletionRefresher
+from mssqlcli.key_bindings import mssqlcli_bindings
+from mssqlcli.mssqlbuffer import MssqlBuffer
+from mssqlcli.mssqlcliclient import MssqlCliClient
+from mssqlcli.mssqlcompleter import MssqlCompleter
+from mssqlcli.mssqlstyle import style_factory
+from mssqlcli.mssqltoolbar import create_toolbar_tokens_func
+from mssqlcli.sqltoolsclient import SqlToolsClient
+from mssqlcli.packages import special
+
+from prompt_toolkit import CommandLineInterface, Application, AbortAction
+from prompt_toolkit.enums import DEFAULT_BUFFER, EditingMode
+from prompt_toolkit.shortcuts import create_prompt_layout, create_eventloop
+from prompt_toolkit.buffer import AcceptAction
+from prompt_toolkit.document import Document
+from prompt_toolkit.filters import Always, HasFocus, IsDone
+from prompt_toolkit.layout.lexers import PygmentsLexer
+from prompt_toolkit.layout.processors import (
+    ConditionalProcessor, HighlightMatchingBracketProcessor)
+from prompt_toolkit.history import FileHistory
+from prompt_toolkit.auto_suggest import AutoSuggestFromHistory
+from pygments.lexers.sql import PostgresLexer
+from pygments.token import Token
+
+
+# Query tuples are used for maintaining history
+MetaQuery = namedtuple(
+    'Query',
+    [
+        'query',                # The entire text of the command
+        'successful',           # True If all subqueries were successful
+        'total_time',           # Time elapsed executing the query
+        'meta_changed',         # True if any subquery executed create/alter/drop
+        'db_changed',           # True if any subquery changed the database
+        'path_changed',         # True if any subquery changed the search path
+        'mutated',              # True if any subquery executed insert/update/delete
+    ])
+MetaQuery.__new__.__defaults__ = ('', False, 0, False, False, False, False)
+
+OutputSettings = namedtuple(
+    'OutputSettings',
+    'table_format dcmlfmt floatfmt missingval expanded max_width case_function'
+)
+OutputSettings.__new__.__defaults__ = (
+    None, None, None, '<null>', False, None, lambda x: x
+)
+
+
+class MssqlCli(object):
+
```

Comment:
Do we have to put this in __init__.py? I'd rather see it in mssqlcli.py

Reply:
Yea putting stuff in init.py is misleading IMO. Agree with Brian

---
In reply to: [171445777](https://github.com/dbcli/mssql-cli/pull/173#discussion_r171445777) [](ancestors = 171445777)